src/deprecated.jl

```diff
@@ -1,9 +1,4 @@
 using Base: @deprecate
 
 # BEGIN TimeZones 1.0 deprecations
```

Comment:
We probably want to get rid of these comments too

Reply:
I usually leave these in for when I add new deprecations. I'll note that these should have been labeled 0.11 which was an error on my part in https://github.com/JuliaTime/TimeZones.jl/pull/231

Reply:
Ah that makes more sense